### ISSUE TITLE
is it possible to translate the button?

### ISSUE THREAD

**Issue by wiwengweng** [mstratman/jQuery-Smart-Wizard#118]
Hi, I am using smart wizard with python flask, and want to translate the button. My pages and py file is translated using flask-babel, so I would like to translate the buttons inside smart wizard.

**Comment by jbrulmans** [comment]
You can use the **lang** tag, when initializing the wizard.
Like in the example below:

    $('#smartwizard').smartWizard({
                lang: {
                    next: 'Next', 
                    previous: 'Previous'
                }
    }

**Comment by wiwengweng** [comment]
@jbrulmans I found that the code snippet is not working for my js lib. Maybe the reason is the js lib version?? But found the following code takes effect when initialized, and hope this will help others:

````
        $('#wizard1').smartWizard({
            transitionEffect: 'fade',
            onFinish: onFinishCallback,
            onLeaveStep: leaveAStepCallback,

        labelNext:'ddd',
        labelPrevious:'fff',
        labelFinish:'ggg',

        });
````

**Comment by jbrulmans** [comment]
@wiwengweng Indeed, you are probably running version 3.x, which used the tags you mentioned.
Documentation for version 3 can be found [here](https://github.com/mstratman/jQuery-Smart-Wizard). I guess this can be closed.

**Comment by wiwengweng** [comment]
ok, thank you very much.
